crates/biome_configuration/src/javascript/mod.rs

```diff
@@ -44,6 +44,20 @@ pub struct JsConfiguration {
     #[bpaf(hide)]
     #[serde(skip_serializing_if = "Option::is_none")]
     pub jsx_runtime: Option<JsxRuntime>,
+
+    /// The JSX factory function to use when using the classic JSX runtime.
+    /// This is read from tsconfig.json's compilerOptions.jsxFactory.
+    /// Only relevant when jsx_runtime is ReactClassic.
```

Comment:
I'm not sure I understand. If this value is read from `tsconfig.json`, why do we add it to `biome.json`?  This comment is valid for the other configuration too